admin/nodes/views.py

```diff
@@ -461,7 +462,7 @@ def delete(self, request, *args, **kwargs):
             message='Node Reindexed (SHARE): {}'.format(node._id),
             action_flag=REINDEX_SHARE
         )
-        if isinstance(node, Node):
+        if isinstance(node, (Node, Registration)):
```

Comment:
Drive-by: I was using the reindex button and noticed it was 502ing for registrations and this fixes it.